src/Grphp/Client/Interceptors/Base.php

```diff
@@ -67,7 +67,7 @@ public function getRequest(): Message
      * @param Message $request
      * @return void
      */
-    public function setRequest(&$request)
+    public function setRequest($request): void
```

Comment:
My secondary intention was to clean-up pass by reference instances. They are unnecessary and in some cases confusing.